### ISSUE TITLE
Identity UI uses heading elements inconsistently

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#25674]
This was reported as an accessibility issue. See details at https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1162883

In short, we should use HTML heading elements in order, starting from `<h1>`, then `<h2>`, etc., without gaps in the sequence. There are many parts of the identity UI that don't do this. For example the "my profile" screen has `h2` then `h4`.

**Comment by blowdart** [comment]
The [linked document](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html) doesn't say anything about heading levels not being skippable or without gaps, and does not seem to be an immediate accessibility violation. As there's no time ever set aside for identity UI this is going to be backlogged, unless you feel it is blocking,

**Comment by SteveSandersonMS** [comment]
@blowdart I know it's not an authoritative rule, but this [W3C page about accessible headings](https://www.w3.org/WAI/tutorials/page-structure/headings/) specifically gives the following example of what not to do:

> Make sure that a &lt;h2&gt; is not followed directly by an &lt;h4&gt;, for example

It may be fine to leave this on the backlog but @mkArtakMSFT has more context on that.

**Comment by blowdart** [comment]
It's as much a case of time, and the cascading changes into templates. @mkArtakMSFT would need to talk with @ajcvickers about Hao's time here.

**Comment by mkArtakMSFT** [comment]
Thanks for investigation, @SteveSandersonMS.
I've resolved the related AzDO issue indicating that we will be addressing this post 5.0.

**Comment by HaoK** [comment]
So just to clarify the change we want to make eventually while things are fresh, we just want to sequentially step up or down?  i.e. h1 h2 h3 h4, and always starting at h1?

**Comment by SteveSandersonMS** [comment]
Yes, that’s what I understand the requirement to be. However I’m still open to the possibility that this is a bogus requirement, as the W3C doc stating it doesn’t give any justification or evidence for the need.

**Comment by HaoK** [comment]
Keeping open until fix is ported to scaffolding

**Comment by HaoK** [comment]
PR for scaffolding is up here https://github.com/dotnet/Scaffolding/pull/1502
